src/snowflake/snowpark/mock/_functions.py

```diff
@@ -17,6 +17,7 @@
 import pytz
 
 import snowflake.snowpark
+from snowflake.connector.options import pandas
```

Comment:
Thanks for fixing this, ideally we can enforce this with some linting in the precommit.